### DISCUSSION TITLE
[object-hash] is outdated

### DISCUSSION THREAD

**Discussion by gerardolima** [DefinitelyTyped/DefinitelyTyped#63297] [Issues with a @types package]
Current version of `object-hash` is `3.0.0` ([npm](https://www.npmjs.com/package/object-hash), [repo](https://github.com/puleos/object-hash)). However, latest version of `@types/object-hash` is compatible to `object-hash@2.2` ([npm](https://www.npmjs.com/package/@types/object-hash), [repo](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/object-hash/index.d.ts)). Are there plans to update the type definitions?

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
@gerardolima there are no changes in API in 3.0, so you can continue to use @types/object-hash v2.x

**Reply by gerardolima** [reply]
Thank you for the information, @Semigradsky, but major semver updates implies changes in interface and this might lead to other developers to distrust these types. Would you consider bumping their version?

**Reply by Semigradsky** [reply]
Developers should always read CHANGELOGs 😄

Feel free to send PR to bump version, if you think that it is needed.
